crates/chia-consensus/src/fast_forward.rs

```diff
@@ -1,43 +1,16 @@
 use crate::error::{Error, Result};
 use chia_protocol::Bytes32;
 use chia_protocol::Coin;
+use chia_wallet::singleton::SingletonArgs;
+use chia_wallet::singleton::SingletonSolution;
+use chia_wallet::singleton::SingletonStruct;
 use chia_wallet::singleton::SINGLETON_TOP_LAYER_PUZZLE_HASH;
-use clvm_traits::{FromClvm, ToClvm};
+use clvm_traits::FromClvm;
+use clvm_traits::ToClvm;
```

Comment:
it's not clear that this is an improvement

Reply:
Fixed